appdatabase/migrations/sql/1647868109_add_waku_messages_table.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE waku_messages (
```

Comment:
You'll likely will be querying these messages by `{timestamp}-{hash}`, so it might be worth thinking whether you want to create an index now, otherwise if the collection grows large it means that the migration might take a long time (and therefore users might be thinking that the app is hanging at login), but up to you